pkg/alertmanager/alertmanager.go

```diff
@@ -220,8 +220,12 @@ func New(cfg *Config, reg *prometheus.Registry) (*Alertmanager, error) {
 	am.silences, err = silence.New(silence.Options{
 		SnapshotFile: silencesFile,
 		Retention:    cfg.Retention,
-		Logger:       log.With(am.logger, "component", "silences"),
-		Metrics:      am.registry,
+		Limits: silence.Limits{
+			MaxSilences:        cfg.Limits.AlertmanagerMaxSilencesCount(cfg.UserID),
+			MaxPerSilenceBytes: cfg.Limits.AlertmanagerMaxSilenceSizeBytes(cfg.UserID),
```

Comment:
`max-per-silence-bytes` makes this limit a lot easier to understand (compared to `max-silence-size-bytes`). WDYT about using max _per_ silence bytes everywhere? OK too if you'd like to keep it as-is for consistency with other limits.

Reply:
I would like to keep it consistent with other limits 🙂